Sources/Extensions/CollectionViewManager+UICollectionViewDataSource.swift

```diff
@@ -26,14 +26,14 @@ extension CollectionViewManager: UICollectionViewDataSource {
     open func collectionView(_ collectionView: UICollectionView,
                              viewForSupplementaryElementOfKind kind: String,
                              at indexPath: IndexPath) -> UICollectionReusableView {
+        //TODO: Add same `configure` logic as with cell items
```

Comment:
With Swift 4.2 looks better to rewrite this TODO to `#warning("TODO: Add same `configure` logic as with cell items")`, there are fewer chances to forget about it.

Reply:
It shouldn't become warning just because of lack of Xcode features 🤔